src/components/ProbeCard/ProbeCard.tsx

```diff
@@ -1,59 +1,92 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { GrafanaTheme2 } from '@grafana/data';
-import { Badge, Button, useStyles2 } from '@grafana/ui';
-import { css, cx } from '@emotion/css';
+import { Card, Link, LinkButton, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
 
-import { type Label, type Probe, ROUTES } from 'types';
+import { type ExtendedProbe, type Label, ROUTES } from 'types';
 import { useCanEditProbe } from 'hooks/useCanEditProbe';
-import { Card } from 'components/Card';
 import { SuccessRateGaugeProbe } from 'components/Gauges';
 import { getRoute } from 'components/Routing.utils';
 
-export const ProbeCard = ({ probe }: { probe: Probe }) => {
+import { ProbeUsageLink } from '../ProbeUsageLink';
+import { ProbeDisabledCapabilities } from './ProbeDisabledCapabilities';
+import { ProbeLabels } from './ProbeLabels';
+import { ProbeStatus } from './ProbeStatus';
+
+export const ProbeCard = ({ probe }: { probe: ExtendedProbe }) => {
   const canEdit = useCanEditProbe(probe);
-  const [isFocused, setIsFocused] = useState(false);
-  const onlineTxt = probe.online ? 'Online' : 'Offline';
-  const onlineIcon = probe.online ? 'heart' : 'heart-break';
-  const color = probe.online ? 'green' : 'red';
-  const probeTypeText = probe.public ? 'Public' : 'Private';
-  const probeTypeIcon = probe.public ? 'cloud' : 'lock';
-  const styles = useStyles2(getStyles);
-  const href = `${getRoute(ROUTES.EditProbe)}/${probe.id}`;
+  const probeEditHref = `${getRoute(ROUTES.EditProbe)}/${probe.id}`;
   const labelsString = labelsToString(probe.labels);
+  const styles2 = useStyles2(getStyles2);
 
   return (
-    <Card className={styles.card} href={href}>
-      <div className={styles.cardContent}>
+    <Card>
+      <Card.Heading>
```

Comment:
I am torn on moving back to using @grafana/ui's `Card` component. IMO they have painted themselves into a corner with how prescriptive it is and it is doing way more work than what I would expect a Design System's card to provide.

I think you've done a good job with what they provide here but I do hate that the heading is a h2 regardless as I'd flag this as an accessibility issue on this page when doing an audit. We could probably add props to the `Card.Heading` upstream in grafana/grafana to fix this at a later date so can leave it but just want to flag why I made a local `Card` component in the first place.